index.html

```diff
@@ -400,6 +400,21 @@ <h3>Timeline</h3>
 
 			<section id="success-criteria">
 				<h2>Success Criteria</h2>
+				
+				<p>The EPUB 3 Working Group’s work is considered a success 
+				if all of the following conditions are met:</p>
+				
+				<ol>
+					<li>There are multiple, independent, interoperable implementations 
+					of EPUB 3.X that are widely used.</li>
+					<li>Existing content is supported. All conforming EPUB 3.2 publications should remain conformant to EPUB 3.X.</li>
+					
+					<li>Changes to the global ecosystem 
```

Comment:
Shouldn't "to the global ecosystem" be "that affect the global ecosystem" here?

We're not actually changing the ecosystem itself.

Reply:
Fixed in 606b31a61287930fedcf68d9cb5383b9275e466d